### DISCUSSION TITLE
Firewalld & Docker & Nftables

### DISCUSSION THREAD

**Discussion by cjs1976** [firewalld/firewalld#1320] [Q&A]
Hi, I am new to firewalld and docker and I'm struggling to find something easy and that doesn`t look like a hack. Perhaps someone here can please help me with an easy and clear description as to how can I protect my Docker containers with firewalld? Thanks in advance!

**Comment by erig0** [comment]
I need more information about what you mean by "protect docker containers with firewalld". Unless the container has published a port then no traffic incoming connections will be allowed to the container.

**Comment by erig0** [comment]
Here is a post about strictly filtering docker containers with firewalld. Perhaps that's useful to you.

https://firewalld.org/2024/04/strictly-filtering-docker-containers
